style.css

```diff
@@ -0,0 +1,136 @@
+html {
+  box-sizing: border-box;
+  font-size: 16px;
+  font-family: 'Inter', sans-serif;
+  scroll-behavior: smooth;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+body {
+  background-color: #fff;
+  margin: 0 auto;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 27px;
+  color: #acb7c3;
+}
+
+.hamburger {
+  display: block;
+  cursor: pointer;
+}
+
+.slide {
+  display: block;
+  height: 3px;
+  width: 25px;
+  margin: 5px auto;
+  background-color: #acb7c3;
+}
```

Comment:
- As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Use%20responsive%20values%20in%20your%20CSS%20rules%2C%20like%20percentages%20(i.e.%20width%3A%2080%25)%2C%20instead%20of%20pixels), I think it would be better to use responsive Css values instead of pixels. Please Kindly change it. You can use `%`, `vh`, `vw` or even `rem` as they are responsive css values

Reply:
This code is for the hamburger slides, I read that it's better to use pixels in this case.
![image](https://user-images.githubusercontent.com/120120777/235270487-f7f3bf49-2a9e-416d-9c89-77e197e60bf0.png)